Update project to prepare for new direction: SJ++ (#7)
AY2324S1-CS2103T-W08-3/tp: README.md

## SteveJobs++ (SJ++)

[![CI Status](https://github.com/AY2324S1-CS2103T-W08-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-W08-3/tp/actions)

> SJ++ is a desktop application helps manage job applications by allowing you to easily record organization and recruiter contacts, plus note down information about your applications. It is optimized for CLI use, so you can operate the application with only your keyboard.

Reviewer: Minor phrasing but should be "... desktop application **that** helps **to** manage..."

> SJ++ is a desktop application helps manage job applications by allowing you to easily record organization and recruiter contacts, plus note down information about your applications. It is optimized for CLI use, so you can operate the application with only your keyboard.

Author: ```suggestion
SJ++ is a desktop application that helps you manage your job applications by easily recording organization and recruiter contacts, plus note down information about your applications. It is optimized for CLI use, so you can operate the application with only your keyboard.
```
